STANDARDS.md

```diff
@@ -1,16 +1,66 @@
-# Dataset Standards
 
-This file outlines the standards and requirements needed for submitting a dataset to QCArchive.
+This file outlines the standards and data requirements needed for submitting a dataset to QCArchive.
 This ensures that we have a consistent data model for downstream processes.
 
-## Required fields 
+STANDARDS version: 3-DRAFT (2020-11-13)
+
+We distinguish between standards for the datasets (i.e. the actual data), and the standards for training force fields.
 
-Current list:
-* Ensure all submissions have cmiles, most important are mapped hydrogen smiles
-* Ensure the WBO is requested for all submissions, this should be included in the scf properties list using the flag `wiberg_lowdin_indices`
+# Dataset Standards
 
-## Best practices
-* If any calculations are to be redone from another collection, re-use the old input (coordinates, atom ordering etc) as this will avoid running the calculation again and will just create new references in the database to the old results and should help keep the cost of the calculations down.  
+## Each molecule must have the following information:
+- Canonical isomeric explict hydrogen mapped SMILES
+- Provenence info of SMILES generation (NEW)
+- Coordinates
+- Total Charge
+- Coordinates must be in CMILES order (discuss)
+
+## Each dataset must have the following information:
+- Name
+- Version (NEW)
+- A short description
+- A long description
+- A link/URL/reference pointing to the provenence to reproduce dataset (e.g. the GH submission folder)
+- A changelog (NEW)
+- A github submitter username
+- The name of the person who selected/sourced the molecules (NEW)
```

Comment:
I agree that this is very useful information along with the changelog but is it not better to have this kind of detailed information in the README that the long description URL points to? I just want to make sure users don't have to repeat themself when making datasets making the process move involved as we are trying to make this as easy as possible.

Reply:
I am slightly tending towards having QCA as the primary source of data. I fear there would be a disconnect by storing informative pieces somewhere else. But, this could definitely be something that is voted on. 

In terms of technical aspects, I presume much of this (e.g. copying metadata for a new dataset) can be automated. For example, having e.g. QCSubmit have a "changes" field would just be some string users input, and the implementation can construct the full changelog. I think the tough issue is knowing which QCSubmit fields need to be filled out, but we can easily make a list/template notebook of this once these standards are settled.